src/haxelib/client/Main.hx

```diff
@@ -244,7 +244,10 @@ class Main {
 	}
 
 	function version() {
-		print(VERSION_LONG);
+		Sys.println('Haxelib\'s version: ${VERSION_LONG}');
+
+		if ( paramOpt() != null )
+			Sys.println('! WARNING: "version" parameter only shows the version of the "haxelib"');
```

Comment:
That's a misleading error.
What if a user typed `haxelib version --verbose`?
The message should tell exactly, what's the problem. And the problem is invalid amound of arguments.

Reply:
I can't get the difference with tag `--verbose`.

I think the only usage is `haxelib version`, isn't it?

Reply:
It's just an example of a case when proposed error message would be misleading.